spec/support/billing_spec_helper.rb

```diff
@@ -211,6 +211,173 @@ def billing_addon_usage_response_body(attributes = {})
       }.deep_merge(attributes)
     end
 
+    def billing_v2_subscription_response_body(attributes={})
```

Comment:
if billing related and not on master - don't merge

Reply:
Duplicated. Solved